### DISCUSSION TITLE
Hyperlinks in SVGs

### DISCUSSION THREAD

**Discussion by schlegelp** [datapane/datapane#93] [Help]
Hi! I'm generating matplotlib plots with hyperlinks. Saving the figures as SVG and opening them in a browser works. I would love to embed those figures in a datapane report but that unfortunately kills the hyperlinks. I've tried adding the figure directly as `datapane.Plot` or from disk as `datapane.File(file='plot.svg')`. Grateful for any suggestions!

Here's a minimal example:
```Python
import matplotlib.pyplot as plt 
import datapane as dp 

fig, ax = plt.subplots()
ax.plot([0, 1], [0, 1])
ax.scatter([1, 2, 3], [4, 5, 6])

# Text with clicky hyperlink
text = ax.annotate("Link", xy=(2,5), xytext=(2.2,5.5),
                    url='http://matplotlib.org', 
                    bbox=dict(color='w', alpha=1e-6, url='http://matplotlib.org'))

# Add plot 
plot = dp.Plot(fig)

# As file 
plt.savefig('clicky_plot.svg')
file = dp.File(file='clicky_plot.svg')

report = dp.Report(plot, file)
report.save('datapane_test.html', open=True)
```

**Comment by mands** [comment]
Thanks for raising - we'll look into this - I'm unsure if this is an issue with how we generate teh SVG, or potentially how we render it in the browser due to sandboxing. Hope to have a fix/workaround soon
